integration-tests/multinode/multitx_test.py

```diff
@@ -0,0 +1,73 @@
+import os
+import time
+from chainrpc import RPC
+from common import (
+    UnixStreamXMLRPCClient, wait_for_validators, stop_node,
+    wait_for_tx, latest_block_height
+)
+
+'''
+Env:
+- Two nodes, each with half power
+
+Procedure:
+- stop node1
+- send tx to node0
+- start node1
+- repeat above
+- test wallet sync
+'''
+
+
+BASE_PORT = int(os.environ.get('BASE_PORT', 25560))
+supervisor = UnixStreamXMLRPCClient('data/supervisor.sock')
+rpc = RPC(BASE_PORT)
+
+print('Wait for 2 validators online')
+wait_for_validators(rpc, 2)
+
+os.environ['ENCKEY'] = rpc.wallet.enckey()
+
+print('Prepare node0 transfer addresses')
+unbonded = rpc.address.list()[1]
+transfer1 = rpc.address.list(type='transfer')[0]
+
+time.sleep(3)  # wait for at least one block, FIXME remove after #828 fixed
+txid = rpc.staking.withdraw_all_unbonded(unbonded, transfer1)
+wait_for_tx(rpc, txid)
+rpc.wallet.sync()
+
+addresses = [rpc.address.create(type='transfer') for i in range(10)]
+amount = 100000000
+for addr in addresses:
+    txid = rpc.wallet.send(addr, amount)
+    wait_for_tx(rpc, txid)
+    rpc.wallet.sync()
+
+print('Stop node1')
+stop_node(supervisor, 'node1')
+
+last_height = latest_block_height(rpc)
+
+print('Send multiple tx')
+pending_txs = [rpc.wallet.send(transfer1, amount) for _ in addresses]
```

Comment:
is `rpc.wallet.send` async?
It seems there needs to be some "wait", otherwise node1 may be started up before everything else?

Reply:
> is `rpc.wallet.send` async?
> It seems there needs to be some "wait", otherwise node1 may be started up before everything else?

Even if it's async, the tx should be processed eventually? Maybe `wait_for_tx` need some more timeout?

Reply:
Changed a little bit, let's see how that works?

Reply:
> Even if it's async, the tx should be processed eventually? 

yes, but my guess is that the goal is here is that these TXs end up in mempool while node1 is down, so that they all get picked up for the same block. If it starts up before that, they'll get picked up immediately for different blocks

Reply:
> > Even if it's async, the tx should be processed eventually?
> 
> yes, but my guess is that the goal is here is that these TXs end up in mempool while node1 is down, so that they all get picked up for the same block. If node1 starts up before that / while they are being submitted, they may get picked up immediately for different blocks

Yes, I added 1-second sleep for this.